src/main/java/org/opensearchmetrics/lambda/GithubEventsLambda.java

```diff
@@ -0,0 +1,89 @@
+package org.opensearchmetrics.lambda;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.opensearchmetrics.dagger.DaggerServiceComponent;
+import org.opensearchmetrics.dagger.ServiceComponent;
+import org.opensearchmetrics.model.event.EventData;
+import org.opensearchmetrics.util.OpenSearchUtil;
+import org.opensearchmetrics.util.S3Util;
+import software.amazon.awssdk.core.ResponseInputStream;
+import software.amazon.awssdk.services.s3.model.GetObjectResponse;
+
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+public class GithubEventsLambda extends AbstractBaseLambda {
+    private final String[] eventsToIndex = {"issues.opened",
+            "issues.closed",
+            "issues.labeled",
+            "issues.unlabeled",
+            "issues.transferred",
+            "issues.assigned",
+            "issue_comment.created",
+            "pull_request.closed",
+            "pull_request.opened",
+            "pull_request.labeled",
+            "pull_request.unlabeled",
+            "pull_request.assigned",
+            "pull_request_review.submitted",
+            "pull_request_review_comment.created",
+            "gollum"};
+    private static final ServiceComponent COMPONENT = DaggerServiceComponent.create();
+    private final OpenSearchUtil openSearchUtil;
+    private final S3Util s3Util;
+    private final ObjectMapper mapper;
+
+    public GithubEventsLambda() {
+        this(COMPONENT.getOpenSearchUtil(), COMPONENT.getS3Util(), COMPONENT.getObjectMapper());
+    }
+
+    @VisibleForTesting
+    GithubEventsLambda(@NonNull OpenSearchUtil openSearchUtil, @NonNull S3Util s3Util, @NonNull ObjectMapper mapper) {
+        this.openSearchUtil = openSearchUtil;
+        this.s3Util = s3Util;
+        this.mapper = mapper;
+    }
+
+    @Override
+    public Void handleRequest(Void input, Context context) {
+        final String yesterday = LocalDate.now(ZoneOffset.UTC).minus(1, ChronoUnit.DAYS).toString(); // yyyy-MM-dd
```

Comment:
Please change `yesterday` to something like `collectionStartTime` and add comments that will default to yesterday.